projects/kit/pipes/extract-country-code/extract-country-code.pipe.ts

```diff
@@ -34,9 +35,10 @@ export class TuiExtractCountryCodePipe implements PipeTransform {
             const matched =
                 ruCodeTest ||
                 (value.startsWith(
-                    this.isoToCountryCodePipe
-                        .transform(countryIsoCode)
-                        .replace(CHAR_PLUS, ``),
+                    tuiIsoToCountryCode(this.countriesMasks, countryIsoCode).replace(
```

Comment:
why not from options `this.options.isoToCountryCode`

Reply:
pipe does not have options